spec/requests/answers_spec.rb

```diff
@@ -1,7 +1,89 @@
 require 'rails_helper'
 
 RSpec.describe "Answers", type: :request do
+  let(:user) do
+    User.create alias: 'test', email: 'test@testing.com', password: '123456', password_confirmation: '123456'
+  end
+  let(:question) do
+    Question.create post: 'Is this toast burnt?', img_url: 'https://live.staticflickr.com/298/31930376435_7d1dcc40ef_b.jpg', user_id: user.id
+  end
+
+  # -----index-----
   describe "GET /index" do
-    pending "add some examples (or delete) #{__FILE__}"
+    it "gets a list of answers" do
+      Answer.create comment: "Time to move on", question_id: question.id, user_id: user.id
+      
+      get '/answers'
+
+      answer = JSON.parse(response.body)
+      expect(response).to have_http_status(200)
+      expect(answer.length).to eq 1  
+    end
+  end
+
+  # -----create-----
+  describe "POST /create" do
+    it "creates a new answer" do
+      answer_params = {
```

Comment:
Optional, but since you're using `answer_params` in multiple places, you could extract it to a `let` as well:
```ruby
  let(:answer_params) do
    {
      answer: {
        comment: "Time to move on",
        question_id: question.id,
        user_id: user.id
      }
    }
  end
```